### ISSUE TITLE
is_address_rom only returns true for the first 8191 bytes

### ISSUE THREAD

**Issue by iosabi** [raspberrypi/pico-bootrom#10]
[`is_address_rom`](https://github.com/raspberrypi/pico-bootrom/blob/master/bootrom/usb_boot_device.h#L31) only returns true for addresses between 0 and 8191 inclusive, however we ROM area is 16 KiB. This causes reads to the bootrom area (with READ commands from the PICOBOOT interface) to fail if we try to read anything in the 8191 to 16384 range. Note that it is not possible to read address 8191 since the READ command checks for [`is_address_rom(address + size)`](https://github.com/raspberrypi/pico-bootrom/blob/master/bootrom/async_task.c#L171).

The function should check `<= 16384` instead of `< 8192`.

A possible workaround is to use EXEC commands to copy the ROM area to RAM and then READ from RAM.

Just FYI if somebody runs into this problem.

**Comment by kilograham** [comment]
Yup - picotool works around this problem using the EXEC method if you try to download the bootrom.
